.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,35 @@
+name: Bug Report
+description: Something not behaving the way it should? Let us know!
+title: "Bug: <Title>"
+labels: [bug, needs triage]
```

Comment:
The repo does not have a `needs triage` label, which would likely cause this to not be applied to new issues.